### PR TITLE
Filter the uploads table to only show uploads that created or updated…

### LINKED ISSUE #907: Upload search functionality returns no results
Curator portal: Production environment 
User journey: Checking an upload and want to search for all the cases created and run some spot checks

Issue: Clicked on the number of new cases created, this link takes you to the linelist tab and populates the search with the upload ID.  No line list results are shown for search 

uploadid:5f485eff867f293c4ec809d8

### PR DESCRIPTION
**Pull request by allysonjp715**
… cases

Fixes #907

### CODE REVIEW

File: verification/curator-service/api/openapi/openapi.yaml [hunk 1]
```diff
@@ -242,6 +242,15 @@ paths:
             minimum: 1
             maximum: 100
             default: 10
+        - name: changes_only
```
Comment (attwad) [inline verification/curator-service/api/openapi/openapi.yaml:245]: nit but we haven't used _ as separator usually, we kept to camelCase for properties.

File: verification/curator-service/ui/cypress/integration/components/UploadsTableTest.spec.ts [hunk 2]
```diff
@@ -32,6 +44,13 @@ describe('Uploads table', function () {
         cy.contains('SUCCESS');
         cy.contains('2');
         cy.contains('0');
+        cy.contains('5ef8e943dfe6e00030892d60');
+        cy.contains('2020-01-03');
+        cy.contains('SUCCESS');
```
Comment (attwad) [inline verification/curator-service/ui/cypress/integration/components/UploadsTableTest.spec.ts:49]: this is a dupe

**Review by attwad — APPROVED** [review]

File: verification/curator-service/api/openapi/openapi.yaml (hunk 1 above)
Comment (allysonjp715) [reply verification/curator-service/api/openapi/openapi.yaml:245]: I saw more uses of _ than camelCase in query parameters, e.g. access_token, validate_only, num_cases.

File: verification/curator-service/ui/cypress/integration/components/UploadsTableTest.spec.ts (hunk 2 above)
Comment (allysonjp715) [reply verification/curator-service/ui/cypress/integration/components/UploadsTableTest.spec.ts]: Done.

File: verification/curator-service/api/openapi/openapi.yaml (hunk 1 above)
Comment (attwad) [reply verification/curator-service/api/openapi/openapi.yaml:245]: oh, ok then.. I was thinking everything in our case data is camelCase, but request params it seems not... no big deal.

**Pull request merged**